--- conanfile.py.org    2024-09-16 17:00:58.495031078 +0200
+++ conanfile.py        2024-09-16 16:59:11.205533912 +0200
@@ -1,7 +1,7 @@
 from conan import ConanFile
 from conan.tools.build import cross_building
 from conan.tools.env import VirtualBuildEnv
-from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir, save
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir, save, replace_in_file
 from conan.tools.gnu import Autotools, AutotoolsToolchain
 from conan.tools.layout import basic_layout
 from conan.tools.microsoft import is_msvc, unix_path
@@ -112,6 +112,7 @@
         self._patch_sources()
         autotools = Autotools(self)
         autotools.configure()
+        replace_in_file(self, os.path.join(self.build_folder, 'Makefile'), search="SUBDIRS = . examples lib src doc checks po tests", replace="SUBDIRS = . examples lib src checks po tests")
         autotools.make()
 
     def package(self):
(python39env) wdobbe@dev-linux-bs03:~/.conan2/p/m4512cc8aabbc4c/e> vi conanfile.py
(python39env) wdobbe@dev-linux-bs03:~/.conan2/p/m4512cc8aabbc4c/e> 
(python39env) wdobbe@dev-linux-bs03:~/.conan2/p/m4512cc8aabbc4c/e> 
(python39env) wdobbe@dev-linux-bs03:~/.conan2/p/m4512cc8aabbc4c/e> 
(python39env) wdobbe@dev-linux-bs03:~/.conan2/p/m4512cc8aabbc4c/e> 
(python39env) wdobbe@dev-linux-bs03:~/.conan2/p/m4512cc8aabbc4c/e> diff -u conanfile.py.org conanfile.py
--- conanfile.py.org    2024-09-16 17:00:58.495031078 +0200
+++ conanfile.py        2024-09-16 17:03:15.222036525 +0200
@@ -1,7 +1,7 @@
 from conan import ConanFile
 from conan.tools.build import cross_building
 from conan.tools.env import VirtualBuildEnv
-from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir, save
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir, save, replace_in_file
 from conan.tools.gnu import Autotools, AutotoolsToolchain
 from conan.tools.layout import basic_layout
 from conan.tools.microsoft import is_msvc, unix_path
@@ -112,6 +112,8 @@
         self._patch_sources()
         autotools = Autotools(self)
         autotools.configure()
+        if cross_building(self):
+            replace_in_file(self, os.path.join(self.build_folder, 'Makefile'), search="SUBDIRS = . examples lib src doc checks po tests", replace="SUBDIRS = . examples lib src checks po tests")
         autotools.make()
 
     def package(self):